source/developers/projects/engine/api/access-to-services.rst

```diff
@@ -0,0 +1,41 @@
+:is-up-to-date: True
+
+:orphan:
+
+.. _access-to-services:
+
+==================
+Access to Services
+==================
+
+When developing templates or scripts only a small list of services are available to use, you can expose other
+services with the following steps.
+
+-------------
+Site Services
+-------------
+
+If your site includes includes a custom application context with services, you can make them available updating the
```

Comment:
Typo? Remove extra `includes` and maybe add `by` in the middle of available and updating so the sentence goes like:
If your site includes a custom application context with services, you can make them available by updating the